### PR TITLE
Expand types of servers that respond to ping command

### LINKED ISSUE #4952: Not a change for this PR.  We may want to expand the set of servers this command works with to include scan servers and maybe compactors.  It could have server type and resource group args.
Not a change for this PR.  We may want to expand the set of servers this command works with to include scan servers and maybe compactors.  It could have server type and resource group args.

_Originally posted by @keith-turner in https://github.com/apache/accumulo/pull/4851#discussion_r1788223229_

### PR DESCRIPTION
**Pull request by dlmarion**
Closes #4952

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java [hunk 1]
```diff
@@ -308,10 +308,9 @@ List<ActiveCompaction> getActiveCompactions(ServerId server)
   List<ActiveCompaction> getActiveCompactions() throws AccumuloException, AccumuloSecurityException;
 
   /**
-   * Throws an exception if a tablet server can not be contacted.
+   * Throws an exception if a compactor, scan server, or tablet server can not be contacted.
    *
-   * @param tserver The tablet server address. This should be of the form
-   *        {@code <ip address>:<port>}
+   * @param tserver The server address. This should be of the form {@code <ip address>:<port>}
```
Comment (keith-turner) [inline core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java:313]: Could deprecate this API and add a new `ping(ServerId)` method.

**Review by keith-turner — APPROVED** [review]

File: core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java (hunk 1 above)
Comment (dlmarion) [reply core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java:313]: Added #4964 for this

File: shell/src/main/java/org/apache/accumulo/shell/commands/PingCommand.java [hunk 2]
```diff
@@ -31,31 +31,37 @@
 
 public class PingCommand extends Command {
 
-  private Option tserverOption, disablePaginationOpt;
+  private Option serverOption, tserverOption, disablePaginationOpt;
```
Comment (ddanielr) [inline shell/src/main/java/org/apache/accumulo/shell/commands/PingCommand.java:34]: Tested this with uno. The `-ts` flag option has the same usage as `-s` 
```
root@uno> ping --help
usage: ping [-?] [-np] [-s <server address>] [-ts <tablet server>]
description: ping compactors, scan servers, or tablet servers
  -?,--help                              display this help
  -np,--no-pagination                    disable pagination of output
  -s,--server <server address>           compactor, scan server, or tablet server address to ping
  -ts,--tabletServer <tablet server>     compactor, scan server, or tablet server address to ping
```
I'm able to pass a compactor `host:port` string with either the `-ts` or `-s` option flags and they both work.
```
[lock.ServiceLock] DEBUG: [zlock#1aa7a951-9a79-498c-8f74-69cf3c6f4469#] Ephemeral node /accumulo/e4262575-b573-48cc-b4ae-54c2284dff33/compactors/default/localhost:9133/zlock#1aa7a951-9a79-498c-8f74-69cf3c6f4469#0000000000 created with data: {"descriptors":[{"uuid":"1aa7a951-9a79-498c-8f74-69cf3c6f4469","service":"CLIENT","address":"localhost:9133","group":"default"},{"uuid":"1aa7a951-9a79-498c-8f74-69cf3c6f4469","service":"COMPACTOR","address":"localhost:9133","group":"default"}]}

root@uno> ping -ts localhost:9133
localhost:9133 OK
root@uno> ping -s localhost:9133
localhost:9133 OK
```
Given that, does the `-ts` flag still need to exist or can it just be replaced with `-s`?

**Review by ddanielr — APPROVED** [review]
Tested this with uno.
```
root@uno> ping --help
usage: ping [-?] [-np] [-s <server address>] [-ts <tablet server>]
description: ping compactors, scan servers, or tablet servers
  -?,--help                              display this help
  -np,--no-pagination                    disable pagination of output
  -s,--server <server address>           compactor, scan server, or tablet server address to ping
  -ts,--tabletServer <tablet server>     compactor, scan server, or tablet server address to ping
```
I'm able to pass a compactor `host:port` string with either the `-ts` or `-s` option flags and they both work.
```
[lock.ServiceLock] DEBUG: [zlock#1aa7a951-9a79-498c-8f74-69cf3c6f4469#] Ephemeral node /accumulo/e4262575-b573-48cc-b4ae-54c2284dff33/compactors/default/localhost:9133/zlock#1aa7a951-9a79-498c-8f74-69cf3c6f4469#0000000000 created with data: {"descriptors":[{"uuid":"1aa7a951-9a79-498c-8f74-69cf3c6f4469","service":"CLIENT","address":"localhost:9133","group":"default"},{"uuid":"1aa7a951-9a79-498c-8f74-69cf3c6f4469","service":"COMPACTOR","address":"localhost:9133","group":"default"}]}

root@uno> ping -ts localhost:9133
localhost:9133 OK
root@uno> ping -s localhost:9133
localhost:9133 OK
```

File: core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java (hunk 1 above)
Comment (dlmarion) [reply core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java:313]: See comment at https://github.com/apache/accumulo/pull/4964#issuecomment-2402099388. I'm thinking we might want to leave this as-is.

Comment (dlmarion) [reply core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java:313]: Closed #4964

Comment (dlmarion) [reply core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java:313]: I'm thinking we might want to leave this as-is. Reason being that if a user constructs a ServerId as an argument to this method, then they might expect that this method will use the type or resource group members as part of the implementation. It doesn't it just uses the `host:port`. If the user already has a ServerId, then they can pass `ServerId.toHostPortString` when calling this.

Comment (dlmarion) [reply core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java:313]: Decided to add `ping(ServerId)` in addition to `ping(String)`, in 4c9f311

**Pull request merged**